src/wasm32/ffi.c

```diff
@@ -434,7 +344,7 @@ ffi_call_js, (ffi_cif *cif, ffi_fp fn, void *rvalue, void **avalue),
   }
   stackRestore(cur_stack_ptr);
   stackAlloc(0); // stackAlloc enforces alignment invariants on the stack pointer
-  var result = CALL_FUNCTION_POINTER(fn, args);
+  var result = getWasmTableEntry(fn).apply(null, args);
```

Comment:
Maybe let's put the LOG_DEBUG back in here.

Reply:
Done!